### PR TITLE
[RG-176] Add gtkwave tcl command

### PR DESCRIPTION
**Pull request by skyler-rs**
Note: this PR doesn't require RG-171 to build, but the window won't invoke until RG-171 is merged.

> ### Motivate of the pull request
> - [x] To address an existing issue. If so, please provide a link to the issue: [RG-176]
> - [ ] Breaking new feature. If so, please describe details in the description part.

> ### Describe the technical details
> #### What is currently done? (Provide issue link if applicable)
> - Nothing
>
> #### What does this pull request change?
> - Adds gtkwave tcl command
>   - gtkwave <file> will open a given file

> ### Which part of the code base require a change
> 
> - [x] Library: Main
> - [ ] Plug-in: <Specify the plugin name>
> - [ ] Engine
> - [ ] Documentation
> - [ ] Regression tests
> - [ ] Continous Integration (CI) scripts

### CODE REVIEW

**Review by alaindargelas — CHANGES_REQUESTED** [review]
This is the wrong place to register this command (main.cpp)
You need to do that in the Compiler/ subsystem.

**Comment by skyler-rs** [comment]
> This is the wrong place to register this command (main.cpp)
> You need to do that in the Compiler/ subsystem.

Thanks for catching that, I'll move it over

**Comment by ravic-rs** [comment]
> > This is the wrong place to register this command (main.cpp)
> > You need to do that in the Compiler/ subsystem.
> 
> Thanks for catching that, I'll move it over

The move seems good to me, will wait for Alain's approval.

**Pull request merged**
